GPU/GPUCommonHW.cpp

```diff
@@ -360,7 +360,7 @@ const CommonCommandTableEntry commonCommandTable[] = {
 };
 
 GPUCommonHW::GPUCommonHW(GraphicsContext *gfxCtx, Draw::DrawContext *draw) : GPUCommon(gfxCtx, draw) {
-	memset(cmdInfo_, 0, sizeof(cmdInfo_));
+	memset(&cmdInfo_, 0, sizeof(cmdInfo_));
```

Comment:
same here

Reply:
fix it https://github.com/hrydgard/ppsspp/pull/18971/commits/39af23aefa8517e17b9ade1d4c065c47d50901e7